Feat create cli (#75)
fossology/fossology-python: README.rst

           username: fossy
           Password: 
           token scope. (Either read or write):
           token_scope: write
           ... - WARNING - [foss_cli.py:406] - Create New Config server: ...h
           ... - WARNING - [foss_cli.py:451] - New Config section server: ...

This will get a token from Fossology Server and store it within the local .foss_cli.ini file. 
On subsequent foss_cli calls those values will be reused.

Verbosity of all foss_cli commands could be increased using the -v verbosity option.

    .. code:: bash

        $poetry run foss_cli  -vv config

Runs the same command with verbosity level  2.
It leaves a log-file in directory .foss_cli_results named .foss_cli.log.

This way ALL cmds could be logged.

To  create  a group:

     .. code:: bash

           poetry run foss_cli -vv create_group  AwesomeGroup

The looging again is left in directory .foss_cli_results named .foss_cli.log by default.

To create a a folder:

     .. code:: bash

          poetry run foss_cli -vv create_folder AwesomeFolder \
                   --folder_group AwesomeGroup \
                   --folder_description "Description of AwesomeFolder"

> To upload a source package to the server and initialize a the schedule_jobs command could be used:

Reviewer: ```suggestion
To upload a source package to the server and initialize a scan workflow including report generation, the start_workflow command could be used:
```

    .. code:: bash

>        Usage: foss_cli schedule_jobs [OPTIONS] FILE_NAME

Reviewer: ```suggestion
       Usage: foss_cli start_workflow [OPTIONS] FILE_NAME
```

>        The foss_cli schedule_jobs command.

Reviewer: ```suggestion
       The foss_cli start_workflow command.
```

       Options:
         --folder_name TEXT              The name of the folder to upload to.
         --file_description TEXT         The description of the upload.
         --dry_run / --no_dry_run        Do not upload but show what would be done.
                                         Use -vv to see output.
         --reuse_newest_upload / --no_reuse_newest_upload
                                         Reuse newest upload if available.
         --reuse_newest_job / --no_reuse_newest_job
                                         Reuse newest scheduled job for the upload if
                                         available.
         --report_format TEXT            The name of the reportformat. [dep5,
                                         spdx2,spdxtv,readmeoss,unifiedreport]
         --access_level TEXT             The access level of the
                                         upload.[private,protected,public]
         --help                          Show this message and exit.

To upload a file from the development source:

    .. code:: bash

>         poetry run foss_cli -vv schedule_jobs tests/files/zlib_1.2.11.dfsg-0ubuntu2.debian.tar.xz \

Reviewer: ```suggestion
        poetry run foss_cli -vv start_workflow tests/files/zlib_1.2.11.dfsg-0ubuntu2.debian.tar.xz \
```
